std/parallelism.d

```diff
@@ -47,32 +56,25 @@ void main() {
     // getTerm is evaluated in parallel as needed by
     // TaskPool.reduce.
     //
-    // Timings on an Athlon 64 X2 dual core machine:
+    // Timings on an Intel i5-3450 quad core machine
+    // for n = 1_000_000_000:
     //
-    // TaskPool.reduce:       12.170 s
-    // std.algorithm.reduce:  24.065 s
+    // TaskPool.reduce:       4.011 s
+    // std.algorithm.reduce:  1.067 s
```

Comment:
Oh sorry, I didn't notice this before. Yes indeed the two timings should be swapped. I will submit a PR soon.